wrk/tc_bench.sh

```diff
@@ -1 +1 @@
-wrk -d 10 -t 4 -c 100 -s $(pwd)/wrk/wrk.lua http://localhost:8083/graphql
```

Comment:
I think we can have a common wrk script now since they are exactly the same.

Reply:
Yeah, that's right. Missed it. Updating.